styles/widgets/common/filterBuilder.less

```diff
@@ -86,11 +111,11 @@
         }
     }
 
-    .dx-treeview .dx-treeview-node {
-        padding-left: 0;
+    .dx-scrollable-content {
+        max-height: 300px;
```

Comment:
Move max-height from .dx-scrollable-content selector to .dx-scrollable-container.